UseCases/Sources/HTTPEndToEnd/main.swift

```diff
@@ -57,8 +56,9 @@ httpClient.get(url: "http://localhost:8080").whenComplete { result in
 sleep(20)
 
 try httpClient.syncShutdown()
+try orderServiceChannel.close().wait()
+try storageServiceChannel.close().wait()
```

Comment:
I guess we don't really have to right now since we kill the EL, but this would be a good way if we had any cleanup to do in `channelRemoved` -- but we don't... So consider this optional here I guess